code/modules/busy_space/organizations.dm

```diff
@@ -775,16 +1088,64 @@
+		new /datum/lore/mission("SCG-F", list("emergency resupply", "hazardous material transport", "strategic resupply", "classified"), ATC_FREIGHT),
+		new /datum/lore/mission("SCG-E", list("damage assesment", "data recovery", "classified"), ATC_SALVAGE),
+		new /datum/lore/mission("SCG-D", list("defense", "patrol", "military response", "joint exercise", "classified"), ATC_DEF),
+		new /datum/lore/mission/prebuilt/scientific("SCG-S"),
+		new /datum/lore/mission/prebuilt/diplomatic("SCG-D"), //local zaddat colonies, independent earth nations, internal diplomatic missions to embassies in Sol, abels rest. actual foreign diplomacy happens elsewhere
+		new /datum/lore/mission/prebuilt/medical_response("SCG-M"),
+		new /datum/lore/mission/prebuilt/defense_response("SCG-D"),
+		)
+
+	serviced = list(
+		/datum/lore/system/vir = 10,
+		/datum/lore/system/sol = 25,
+		/datum/lore/system/alpha_centauri = 20,
+		/datum/lore/system/tau_ceti = 20,
+		/datum/lore/system/kess_gendar = 20,
+		/datum/lore/system/saint_columbia = 20,
+		/datum/lore/system/jahans_post = 15,
+		/datum/lore/system/abels_rest = 15,
+		/datum/lore/system/el = 10,
+		/datum/lore/system/raphael = 10,
+		/datum/lore/system/altair = 10,
+		/datum/lore/system/zhu_que = 10,
+		/datum/lore/system/love = 5,
+		/datum/lore/system/isavaus_gamble = 3,
+		/datum/lore/system/whythe = 3, //not in scg space but im sure they can still get at it
+		)
 
-	ship_prefixes = list("SCG-T" = "transportation", "SCG-D" = "diplomatic", "SCG-F" = "freight", "SCG-M" = "medical response")
-	destination_names = list(
-						"Venus",
-						"Earth",
-						"Luna",
-						"Mars",
-						"Titan"
-						)// autogen will add a lot of other places as well.
+//kind of experimental
+//lets SCG do diplomacy but not eg military action in foreign systems
+//in the future, similar splits could allow corps to have 'core' defense assets they dont send on idiotic military actions to Vounna
+//could also be expanded with fluff ala VGA/Sifguard
+/datum/lore/organization/gov/solgov/diplo_corps
+	missions = list(
+		new /datum/lore/mission/prebuilt/diplomatic("SCG-D")
+		)
+	serviced = list(
+		/datum/lore/system/qerrvalis = 15, //capital systems of major interstellar powers
+		/datum/lore/system/rarkajar = 15,
+		/datum/lore/system/new_seoul = 15,
+		/datum/lore/system/relan = 15,
+		/datum/lore/system/casinis_reach = 10, //independent single systems
+		/datum/lore/system/eutopia = 10,
+		/datum/lore/system/phact = 10,
+		/datum/lore/system/natuna = 10,
+		/datum/lore/system/neon_light = 5, // neon light (neon light)
+		/datum/lore/system/exalts_light = 5, //noncapital systems of major interstellar powers
+		/datum/lore/system/vounna = 5,
+		/datum/lore/system/sidhe = 5,
+		/datum/lore/system/new_cairo = 3, //funny bugs
+//		/datum/lore/system/mahimahi = 5, //uncomment these if theyre ever like, detailed enough to have a specific named embassy
+//		/datum/lore/system/kauqxum = 5,
```

Comment:
Should probably just remove these until then. **And don't forget the commas.**

Reply:
done